geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/GeoMesaConfigurator.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2015 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.jobs
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.io.serializer.WritableSerialization
+import org.apache.hadoop.mapreduce.Job
+import org.locationtech.geomesa.jobs.mapreduce.SimpleFeatureSerialization
+
+import scala.collection.JavaConversions._
+
+/**
+ * Common place for setting and getting values out of the mapreduce config
+ */
+object GeoMesaConfigurator {
+
+  private val prefix         = "org.locationtech.geomesa"
+
+  private val dsInParams     = s"$prefix.params.in."
+  private val dsOutParams    = s"$prefix.params.out."
+  private val dsInRegex      = dsInParams.replaceAll("\\.", "\\.") + ".+"
```

Comment:
Yeah, exactly. Plus it adds a '.+' on the end, so it will match anything starting with 'org.locationtech.geomesa.params.in.' Then it's used in the Configuration.getValByRegex method.